libs/domain/picking/src/components/picking-list-card/picking-list-card.component.ts

```diff
@@ -1,16 +1,98 @@
-import { html, LitElement } from 'lit';
-import { property } from 'lit/decorators.js';
+import { resolve } from '@spryker-oryx/di';
+import { PickingListService } from '@spryker-oryx/picking';
+import { html, LitElement, TemplateResult } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
 import type { PickingList } from '../../models';
+import { styles } from './picking-list-card.styles';
 
 export class PickingListCardComponent extends LitElement {
-  @property() pickingList?: PickingList;
+  static styles = styles;
 
-  protected override render(): unknown {
-    return html`<p>
-      PickingList ${this.pickingList?.id} in status ${this.pickingList?.status}
-      with ${this.pickingList?.items.length} item(s)
-    </p>`;
+  protected pickingListService = resolve(PickingListService);
+
+  @property({ type: Object }) pickingList?: PickingList;
+  @state() isDisabled?: boolean;
+
+  protected startPicking(): void {
+    this.isDisabled = true;
+    // TODO: push to customer note page if this.pickingList?.cartNote exists
+    // else
+    this.pickingListService.startPicking(this.pickingList!);
+    // TODO: push to picking-items page
+
+    this.isDisabled = false;
   }
-}
 
-export default PickingListCardComponent;
+  protected showCustomerNote(): void {
+    this.dispatchEvent(
+      new CustomEvent('showCustomerNote', {
+        detail: {
+          customerNote: this.pickingList?.cartNote,
```

Comment:
Do we need to show the note in case we don't have `this.pickingList?.cartNote`?

I believe need to dispatch the event in case if we have something to show